docs/source/markdown/podman-run.1.md

```diff
@@ -1341,10 +1320,11 @@ b
 ## FILES
 
 **/etc/subuid**
+
 **/etc/subgid**
 
 ## SEE ALSO
-subgid(5), subuid(5), libpod.conf(5), systemd.unit(5), setsebool(8), slirp4netns(1), fuse-overlayfs(1)
+**subgid**(5), **subuid**(5), **libpod.conf**(5), **systemd.unit**(5), **setsebool**(8), **slirp4netns**(1), **fuse-overlayfs**(1).
 
 ## HISTORY
 September 2018, updated by Kunal Kushwaha <kushwaha_kunal_v7@lab.ntt.co.jp>
```

Comment:
With the changes you've made, you really should take credit and add yourself as an updater here though.  Your call.

Reply:
In fact I was thinking about the opposite. Maybe it makes sense to remove (or, rather, comment out ) this section entirely. Not to offend all the people who have contributed, but listing these names and dates does not really add any value to a reader (except maybe the bit that says docs were initially adopted from Docker).

But that is a separate discussion.

Reply:
I smell a thumb wrestling match when we meet!